influunt-app/features/support/scripts/insert_mongodb.js

```diff
@@ -0,0 +1,61 @@
+var mongo = require('mongodb');
+var assert = require('assert');
+var Long = require('mongodb').Long;
+var MongoClient = mongo.MongoClient;
+
+var url = 'mongodb://127.0.0.1:27017/influuntdev';
+var ObjectId = mongo.ObjectID;
+var NumberLong = Long.fromString('1479756629334');
+
+var clearDb = function(db, callback) {
+	'use strict';
+   db.collection('trocaPlanosControladores').deleteMany(
+      { '_id': ObjectId('58334b5573835c48f22fb7a0') },
+      function() {
+         console.log('ObjectId 58334b5573835c48f22fb7a0 apagado com sucesso.');
+         callback();
+      }
+   );
+};
+
+var insertDocuments = (db, callback) => {
+		'use strict';
```

Comment:
Se colocar ele primeiro o codacy resulta em dois warnings abaixo

line 25  col 5  'clearDb' was used before it was defined.
line 36  col 5  'insertDocuments' was used before it was defined.

 Eu primeiramente até tinha feito em cima, porém como o codacy reclamou disso coloquei abaixo, e funcionou corretamente